docs/troubleshooting.md

```diff
@@ -172,34 +172,26 @@ token that is required to authenticate with the API server.
 Verify with the following commands:
 
 ```console
-# start a container that contains curl
-$ kubectl run test --image=tutum/curl -- sleep 10000
+# get service IP of master
+$ kubectl get services kuberentes
+NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+kubernetes   10.100.0.1     <none>        443/TCP   1d
 
-# check that container is running
-$ kubectl get pods
-NAME                   READY     STATUS    RESTARTS   AGE
-test-701078429-s5kca   1/1       Running   0          16s
+# start a container that contains curl
+$ kubectl run -i --tty --rm test --image=tutum/curl --restart=Never
 
 # check if secret exists
-$ kubectl exec test-701078429-s5kca -- ls /var/run/secrets/kubernetes.io/serviceaccount/
+root@test:/# ls /var/run/secrets/kubernetes.io/serviceaccount/
 ca.crt
 namespace
 token
 
-# get service IP of master
-$ kubectl get services
-NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
-kubernetes   10.0.0.1     <none>        443/TCP   1d
-
 # check base connectivity from cluster inside
-$ kubectl exec test-701078429-s5kca -- curl -k https://10.0.0.1
+root@test:/# curl -k https://10.100.0.1
```

Comment:
For debugging inside container you don't need the control plane IP. Can you please try with "Kubernetes.default.svc.cluster.local" and skip the service IP get and the curl direct to the IP?

Reply:
Just made this change.  Sorry for the delay.